v60_type_r build fails with RGB enabled

<!-- Provide a general summary of the bug in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Describe the Bug
Unable to build for v60_type_r on latest with RGB support enabled
Attempted with followingghosts and default layout in clean QMK vagrant environment 
<!-- A clear and concise description of what the bug is. -->

## System Information

 - Keyboard:  v60_type_r polestar
   - Revision (if applicable):  
 - Operating system: QMK Dockerfile:latest 
 - AVR GCC version: 4.9.2
<!-- Run `avr-gcc --version` to find this out. -->
 - ARM GCC version: 6.3.1
<!-- Run `arm-none-eabi-gcc --version` to find this out. -->
 - QMK Firmware version: 0.7.44
<!-- Run `git describe --abbrev=0 --tags` to find this out. -->


## Additional Context

<!-- Add any other relevant information about the problem here. -->
Trying to build latest version of QMK for v60_type_r polestar using a clean copy of the vagrant environment in Docker

When I enable RGB, and define `V60_POLESTAR`, the build fails with the following error:
```
Linking: .build/v60_type_r_default.elf                                                              [ERRORS]
 |
 | .build/obj_v60_type_r_default/quantum/backlight/backlight_avr.o: In function `__vector_17':
 | /vagrant/quantum/backlight/backlight_avr.c:259: multiple definition of `__vector_17'
 | .build/obj_v60_type_r_default/keyboards/v60_type_r/v60_type_r.o:/vagrant/keyboards/v60_type_r/v60_type_r.c:155: first defined here
 | collect2: error: ld returned 1 exit status
 |
tmk_core/rules.mk:298: recipe for target '.build/v60_type_r_default.elf' failed
make[1]: *** [.build/v60_type_r_default.elf] Error 1
Make finished with errors
Makefile:555: recipe for target 'v60_type_r:default' failed
make: *** [v60_type_r:default] Error 1
```

--- a/keyboards/v60_type_r/v60_type_r.c
+++ b/keyboards/v60_type_r/v60_type_r.c
@@ -106,35 +106,35 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
 
 
 void rgb_timer_init(void) {
-    /* Timer1 setup */
+    /* Timer3 setup */
     /* CTC mode */
-    TCCR1B |= (1<<WGM12);
-    /* Clock selelct: clk/8 */
-    TCCR1B |= (1<<CS10);
+    TCCR3B |= _BV(WGM32);
+    /* Clock select: clk/8 */
+    TCCR3B |= _BV(CS30);
     /* Set TOP value */
     uint8_t sreg = SREG;
     cli();
-    OCR1AH = (SOFTPWM_LED_TIMER_TOP >> 8) & 0xff;
-    OCR1AL = SOFTPWM_LED_TIMER_TOP & 0xff;
+    OCR3AH = (SOFTPWM_LED_TIMER_TOP >> 8) & 0xFF;
+    OCR3AL = SOFTPWM_LED_TIMER_TOP & 0xFF;
     SREG = sreg;
 
-    // Enable the compare match interrupt on timer 1
-    TIMSK1 |= (1<<OCIE1A);
+    // Enable the compare match interrupt on timer 3
+    TIMSK3 |= _BV(OCIE3A);
 }
 
 void rgb_init(void) {
-    DDRF  |=  (1<<PF6 | 1<<PF5 | 1<<PF4);
-    PORTF |=  (1<<PF6 | 1<<PF5 | 1<<PF4);
+    DDRF  |= (_BV(PF6) | _BV(PF5) | _BV(PF4));
+    PORTF |= (_BV(PF6) | _BV(PF5) | _BV(PF4));
 
     rgb_timer_init();
 }
 
 void set_rgb_pin_on(uint8_t pin) {
-	PORTF &= ~(1<<pin);
+	PORTF &= ~_BV(pin);
 }
 
 void set_rgb_pin_off(uint8_t pin) {
-	PORTF |= (1<<pin);
+	PORTF |= _BV(pin);
 }
 
 void rgblight_set(void) {
@@ -151,7 +151,7 @@ void rgblight_set(void) {
    //  //xprintf("Red: %u, Green: %u, Blue: %u\n", led[0].r, led[0].g, led[0].b);
 }
 
-ISR(TIMER1_COMPA_vect)
+ISR(TIMER3_COMPA_vect)
 {
     static uint8_t pwm = 0;
     pwm++;

